hawc/apps/epi/api.py

```diff
@@ -139,3 +540,21 @@ class ExposureCleanup(CleanupFieldsBaseViewSet):
     serializer_class = serializers.ExposureCleanupFieldsSerializer
     model = models.Exposure
     assessment_filter_args = "study_population__study__assessment"
+
+
+class Metadata(viewsets.ViewSet):
+    def list(self, request):
+        return EpiMetadata.handle_request(request)
+
+    def retrieve(self, request, *args, **kwargs):
```

Comment:
make this dumber; require an assessment. use mixins.RetrieveModelMixin, and just check that assessment.user_can_view_object(request.user). No list view.